js/chat_app/src/redux/slices/chatSlice.js

```diff
@@ -0,0 +1,48 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { chatServices } from '../../services/chatServices';
+
+const initialState = {
+  title: '',
+  chatId: '',
+  members: [],
+  message: []
+};
+
+const chatSlice = createSlice({
+  name: 'chat',
+  initialState,
+  reducers: {
+    setMembers: (state, action) => {
+      state.members = action.payload;
+    }
+  }
+});
+
+export const addChatMembersThunk = createAsyncThunk(
+  'addMembers/api/chat/:chatId/members',
+  async (paramsForAddMembers) => {
+    const { chatId, authToken, members } = paramsForAddMembers;
+    await chatServices.addChatMembers(chatId, authToken, members);
```

Comment:
it's better just write `return chatServices.addChatMembers(chatId, authToken, members);`